.github/workflows/no-prs-to-translation.yml

```
@@ -16,10 +16,19 @@ jobs:
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
-            await github.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: 'We have implemented a new policy of no longer accepting curriculum-related PRs with changes to non-English files. We plan to implement a better way to translate the English challenges in the near future. We will make an announcement when it is ready. For now, please revert your changes to the non-English challenge files. Thank you.'
-            })
-            throw new Error('This PR appears to touch translated challenge files.')
+            if (context.payload.pull_request.head.repo.fork) {
+              github.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: 'We have implemented a new policy of no longer accepting curriculum-related PRs with changes to non-English files. We plan to implement a better way to translate the English challenges in the near future. We will make an announcement when it is ready. For now, please revert your changes to the non-English challenge files. Thank you.'
```

I do think we should change this message to reflect the announcement that will happen soon about the new Chinese and Spanish curriculum translation effort.  We should hold off merging it until the announcement is made.

```suggestion
                body: 'We have implemented a new policy of no longer accepting curriculum-related PRs with changes to non-English files. Please visit https://translate.freecodecamp.org to make translations to the curriculum. Thank you.'
```

---

> We should hold off merging it until the announcement is made.

I don't think we should hold off - I'd rather have this land sooner to avoid accidentally merging changes to a translated file (we did that once already this week).